sunpy/map/mapsequence.py

```diff
@@ -511,3 +511,31 @@ def all_meta(self):
         Return all the meta objects as a list.
         """
         return [m.meta for m in self.maps]
+
+    def save(self, filepath, filetype='auto', **kwargs):
+        """Saves the SunPy Map Sequence object to a file.
+
+        Currently SunPy can only save files in the FITS format.
+
+        Parameters
+        ----------
+        filepath : str
+            Location to save file to.
+            Each map from the sequence will be saved separately.
+        filetype : str
+            'auto' or any supported file extension.
+        kwargs :
+            Any additional keyword arguments are passed to
+            `~sunpy.io.write_file`.
+
+        Notes
+        -----
+        The individual maps are saved as with the specified
+        `filepath` based on {index} specified by the user.
```

Comment:
I feel like this should be in the "filepath" part. This seems to important to be at the bottom.

Reply:
I'm confused as to where exactly you want this part

Reply:
In the filepath doc part under parameters.